memcached.h

```diff
@@ -428,6 +428,7 @@ struct settings {
     unsigned int logger_watcher_buf_size; /* size of logger's per-watcher buffer */
     unsigned int logger_buf_size; /* size of per-thread logger buffer */
     bool drop_privileges;   /* Whether or not to drop unnecessary process privileges */
+    bool no_watch; /* allows watch commands to be dropped */
```

Comment:
why "no_watch" vs "watch_enabled", like flush_enabled?

Reply:
I think I saw a "no_foo" flag elsewhere.  Happy to reverse this if you prefer.  I also noticed that the flush feature was a regular option, not an extended option, but I had already started to write this as an extended option - would you like me to change that or is an extended option fine?

Reply:
hmm I guess the options are kind of inconsistent. Lets make it a top level option if there's an obvious free character? Most of the extendeds are tuning or very rarely used.